### DISCUSSION TITLE
How to validate the json format comply with the XSD and Spec

### DISCUSSION THREAD

**Discussion by figoyan** [enbility/eebus-go#138] [General]
How do you do such validation? Is there a tool or just a manual confirm?

**Comment by DerAndereAndi** [comment]
There is no tool available. So testing is:
- 1st manual
- 2nd against a device with an existing implementation
- 3rd check with the responsible person from the EEBUS initiative.

There has been different interpretations in the past where some vendors implemented something differently and clarification had to go all 3 steps.

**Reply by figoyan** [reply]
Got it, thanks for the advise.
